src/lib/fields/VocabularyField.js

```diff
@@ -17,39 +17,96 @@ export class VocabularyField extends Field {
     deserializedDefault = null,
     serializedDefault = null,
     labelField = 'name',
+    localeFields = [],
```

Comment:
This 'locale' logic might not be necessary if the backend returns the objects properly internationalised. E.g. : 

```
{
  award: {
    title: "Award title already in desired locale" 
}
}
```

This is done e.g. for licenses.